docs/proposals/portable-service-assets.md

```diff
@@ -0,0 +1,84 @@
+# Portable Service Assets
+
+## Introduction
+Service classes in the Service Catalog require a number of UI graphical assets (i.e. service icon, company logo, etc).  For APBs that are sourced from the RHCC, we do not want to reference these externally on a remote (non-RH managed endpoint).  Additionally, we want to be able to handle off-network / disconnected clusters thus making this a truly portable solution.
```

Comment:
first use of APBs in the document, I'd spell it out Ansible Playbook Bundles (APBs). then use APB in the rest of the document.